packages/bng-metric-service/src/helpers/extractors/extraction-config/configuration.js

```diff
@@ -22,12 +23,7 @@ export default {
     try {
       const result = {}
       const currentMetricVersion = _.isEmpty(options.v) ? (process.env.CURRENT_METRIC_VERSION || 'v4.0') : options.v
-      const configFolderPath = `bng-metric-service/src/helpers/extractors/extraction-config/metric/${currentMetricVersion}/`
-      let fullConfigFolderPath = path.resolve('./', `../${configFolderPath}`)
-      /* istanbul ignore else */
-      if (!fs.existsSync(fullConfigFolderPath)) {
-        fullConfigFolderPath = path.resolve(`packages/${configFolderPath}`)
-      }
+      const fullConfigFolderPath = path.join(dirname, 'metric', currentMetricVersion)
```

Comment:
@paulsimonandrews  - This is the main change in this pull request. Most of the changes are associated with versioning to support deployments to cloud infrastructure.